helpdesk_mgmt_fieldservice/models/helpdesk_ticket.py

```diff
@@ -84,7 +84,7 @@ def _compute_all_closed(self):
             ticket.all_orders_closed = True
             if ticket.fsm_order_ids:
                 for order in ticket.fsm_order_ids:
-                    if order.stage_id.name not in ['Closed', 'Cancelled']:
+                    if not order.stage_id.is_closed:
```

Comment:
You could probably collapse the loop into an `any()` call, but that's more of a flavour thing (po-tay-toe, po-tah-toe)

Functionally works. I'd encourage a quick test, but given there's no coverage anyway and it's an older version of Odoo, I'm not sure what the OCA stance is on this... I'd need one of the maintainers to weigh in?

Depending on the level of brokenness, I would perhaps set up a migration to recompute the field, given the change in definition (i.e. to retrospectively fix any history tickets with incorrect values).

Reply:
I'd be more than happy to refactor the method; I remember reading somewhere that there was a strong preference for generating the smallest possible diff in a PR, so I stuck to the existing structure.